pkgdb2/templates/master.html

```diff
@@ -47,8 +47,13 @@
               <option value="retired"
               {% if select=='retired' %} selected {% endif %}>
                   Retired</option>
+              {% for ns in namespaces %}
+              <option value="{{ ns }}"
+              {% if select==ns %} selected {% endif %}>
```

Comment:
select is provided to the template and here, in the options for searching pkgdb, we try to make sure that if you searched for ``packagers`` or ``docker``, the drop-down of the search options is set to what you searched (``packagers`` or ``docker``)